conans/client/generators/__init__.py

```diff
@@ -67,6 +68,7 @@ def __getitem__(self, key):
 registered_generators.add("qbs", QbsGenerator)
 registered_generators.add("scons", SConsGenerator)
 registered_generators.add("visual_studio", VisualStudioGenerator)
+registered_generators.add("msvc", MSVCGenerator)
```

Comment:
Need a different name, `msvc` is the name of the compiler and this is something related to the Visual Studio IDE  projects